tools/mo/unit_tests/mo/utils/ir_reader/ops_test.py

```diff
@@ -55,6 +57,28 @@ def test_interpolate_11(self):
         interpolate_node = graph.get_op_nodes(op="Interpolate")[0]
         self.assertEqual(interpolate_node["version"], "opset11")
 
+    def test_interpolate_11_scales(self):
+        data_shape = [6, 12, 10, 24]
+        data_parameter = opset11.parameter(
+            data_shape, name="Data", dtype=np.float32)
+        interpolate = opset11.interpolate(data_parameter, np.float32(
+            [2., 2.]), "nearest", "scales", axes=np.int32([2, 3]), name="Interpolate_11")
+        model = Model(interpolate, [data_parameter])
+        graph = TestOps.check_graph_can_save(model, 'interpolate_model')
+        interpolate_node = graph.get_op_nodes(op="Interpolate")[0]
+        self.assertEqual(interpolate_node["version"], "opset11")
+
+    def test_interpolate_11_no_axes(self):
+        data_shape = [6, 12, 10, 24]
+        data_parameter = opset11.parameter(
+            data_shape, name="Data", dtype=np.float32)
+        interpolate = opset11.interpolate(data_parameter, np.int32(
+            [6, 12, 20, 48]), "nearest", "sizes", name="Interpolate_11")
```

Comment:
should not we specify new sizes for spatial dimensions according to the specification? By default, it should two sizes.